README.md

```diff
@@ -55,12 +78,24 @@ If you want to run external-snapshotter with higher availability, you can enable
 --leader-election=true
 ```
 
+### Upgrade from Alpha to Beta
+
+The change from Alpha to Beta snapshot APIs is not backward compatible.
+
+If you have already deployed Alpha snapshot APIs and controller and want to upgrade to Beta, you need to do the following:
```

Comment:
You can also set the deletepolicy to retain, delete the alpha objects, and then import them as pre-existing snapshots using the beta APIs

Reply:
Sure.  I can add that.  I actually have that in my first draft of the README but removed them when I submitted the PR because I thought maybe the workflow that @yuxiangqian is working on can better address this.